.npmignore

```diff
@@ -13,3 +14,4 @@ yarn.lock
 # dev scripts
 operations/*.js
 !operations/201*.js
+test/
```

Comment:
this is intentional - `npm explore foo && npm install && npm test` should work for all packages.